docs/whats-new.md

```diff
@@ -4,36 +4,60 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v7.0
 
-Target Release Date: Mar, 2019
-
-### New Effects System
-
-A new effects system is written from the ground up for v7.0. This opens the possibilities for many exciting visual effect features down the road. As a start, we're introducing [LightingEffect](/docs/effects/lighting-effect.md) - an easier, more comprehensive way to control the lighting for your layers.
+Release Date: April 19, 2019
 
 ### New Layer Catalog
 
 <table style="border: 0;" align="center">
   <tbody>
+    <tr>
+      <td>
+        <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/bitmap-layer.png" />
+        <p><i>BitmapLayer</i></p>
+      </td>
+      <td>
+        <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/column-layer.png" />
+        <p><i>ColumnLayer</i></p>
+      </td>
+      <td>
+        <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/tile-layer.gif" />
+        <p><i>TileLayer</i></p>
+      </td>
+    </tr>
     <tr>
       <td>
         <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/s2-layer.png" />
         <p><i>S2Layer</i></p>
       </td>
+      <td>
+        <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/h3-layer.png" />
+        <p><i>H3HexagonLayer</i></p>
+      </td>
+      <td>
+        <img height=200 src="https://raw.github.com/uber-common/deck.gl-data/master/images/whats-new/h3-cluster-layer.png" />
```

Comment:
I am thinking we can make a better demo to replace this one. But I need know where to get the H3 indexes in order to generate the clusters.

Reply:
Is this something we can/should tackle before 7.0?  On one hand, I'm all for making a great impression in our Whats New docs with nice demo images.  On the other hand, we can pretty easily update this later.